rts/idris_main.c

```diff
@@ -3,6 +3,7 @@
 #include "idris_rts.h"
 #include "idris_stats.h"
 
+void _idris__123_runMain_95_0_125_(VM* vm, VAL* oldbase);
```

Comment:
Oh, yes, it's the idris entry point, it's called further down in idris_main.c. I just added a prototype to avoid a warning if it wasn't defined in the same file.